tenable/ad/checker_option/api.py

```diff
@@ -0,0 +1,103 @@
+'''
+Checker Option
+==============
+
+Methods described in this section relate to the the checker option API.
+These methods can be accessed at ``TenableAD.checker_option``.
+
+.. rst-class:: hide-signature
+.. autoclass:: CheckerOptionAPI
+    :members:
+'''
+from typing import List, Dict
+from tenable.ad.checker_option.schema import CheckerOptionSchema
+from tenable.base.endpoint import APIEndpoint
+
+
+class CheckerOptionAPI(APIEndpoint):
+    _schema = CheckerOptionSchema()
+
+    def list(self,
+             profile_id: str,
+             checker_id: str,
+             **kwargs
+             ) -> List[Dict]:
+        '''
+        Retrieves the list of checker-options.
+
+        Args:
+            profile_id (str):
+                The profile instance identifier.
+            checker_id (str):
+                The checker instance identifier.
+            staged (optional, bool):
+                Get only the checker-options that are staged. Accepted
+                values are ``True`` and ``False``. Added checker options
+                are first staged until the profile is commited. The staged
+                profile options are not activated and don't affect
+                yet the IOE and the exposure detection.
+
+        Returns:
+            list:
+                List of checker options.
+
+        Examples:
+            >>> tad.checker_option.list(
+            ...     profile_id='9',
+            ...     checker_id='1',
+            ...     staged=True
+            ...     )
+        '''
+        partial = ('checker_id', 'profile_id', 'codename', 'value',
+                   'value_type', )
+        params = self._schema.dump(self._schema.load(kwargs, partial=partial))
```

Comment:
As your fundamentally just using the schema to check the single value here, wouldn't just setting partial to `True` make more sense?

```suggestion
        params = self._schema.dump(self._schema.load(kwargs, partial=True))
```